client-v2/integration/selectors/index.js

```diff
@@ -34,12 +43,67 @@ export const clipboardDropZone = maybeGetNth(
 export const clipboardItem = maybeGetNth(
   select(CLIPBOARD_SELECTOR, COLLECTION_ITEM_SELECTOR)
 );
-export const clipboardItemTruncatedHeadline = maybeGetNth(
-  select(CLIPBOARD_SELECTOR, COLLECTION_ITEM_SELECTOR, HEADLINE_SELECTOR)
-);
 export const feedItemAddToClipboardHoverButton = maybeGetNth(
   select(FEED_ITEM_SELECTOR, ADD_TO_CLIPBOARD_BUTTON)
 );
+export const clipboardItemHeadline = maybeGetNth(
+  select(CLIPBOARD_SELECTOR, COLLECTION_ITEM_SELECTOR, HEADLINE_SELECTOR)
+);
+export const clipboardItemDeleteButton = maybeGetNth(
+  select(CLIPBOARD_SELECTOR, COLLECTION_ITEM_SELECTOR, DELETE_BUTTON)
+);
+
+// Collections //
+export const collection = maybeGetNth(select(COLLECTION_SELECTOR));
+
+export const allCollectionItems = collectionIndex => {
+  const collectionSelected = collection(collectionIndex);
+  return collectionSelected.find(`[data-testid="${COLLECTION_ITEM_SELECTOR}"]`);
+};
+
+export const allCollectionDropZones = collectionIndex => {
+  const collectionSelected = collection(collectionIndex);
+  return collectionSelected.find(`[data-testid="${DROP_ZONE_SELECTOR}"]`);
+};
+
+export const collectionItem = (collectionIndex, itemIndex = 0) =>
+  allCollectionItems(collectionIndex).nth(itemIndex);
+
+export const collectionDropZone = (collectionIndex, dropZoneIndex = 0) =>
+  allCollectionDropZones(collectionIndex).nth(dropZoneIndex);
+
+export const collectionItemHeadline = (collectionIndex, itemIndex = 0) =>
+  collectionItem(collectionIndex, itemIndex).find(
+    `[data-testid="${HEADLINE_SELECTOR}"]`
```

Comment:
This pattern of using the `[data-testid="${HEADLINE_SELECTOR}"]` doesn't use the clean `select` helper method that is defined here: https://github.com/guardian/facia-tool/blob/master/client-v2/integration/helpers/index.js